src/run_program.rs

```diff
@@ -1165,16 +1165,67 @@ mod tests {
             err: "softfork specified cost mismatch",
         },
 
-        // without the flag to enable the BLS extensions, it's an unknown extension
+        // without the flag to enable the keccak extensions, it's an unknown extension
         RunProgramTest {
-            prg: "(softfork (q . 161) (q . 1) (q . (q . 42)) (q . ()))",
+            prg: "(softfork (q . 161) (q . 2) (q . (q . 42)) (q . ()))",
```

Comment:
shouldn't `42` be `62` here?

Reply:
I don't think this tests the operator, just an arbitrary body to the softfork op (it was here before keccak) @arvidn